data/sql/updates/pending_db_world/rev_1556033550219221000.sql

```diff
@@ -0,0 +1,55 @@
+(2008, 0, 0, 0, 0, 0, 100, 0, 3300, 3300, 15800, 15800, 11, 11976, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Gnarlpine Warrior - In Combat - Cast \'11976\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 2025 (name: Timberling Bark Ripper)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 2025;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 2025);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2025, 0, 0, 0, 0, 0, 50, 0, 3700, 3700, 14100, 14100, 11, 6016, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Timberling Bark Ripper - In Combat - Cast \'6016\'');
+
+
+-- Creating a new SmartAI script for [Creature] ENTRY 1999 (name: Webwood Venomfang)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 1999;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 1999);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(1999, 0, 0, 0, 0, 0, 100, 0, 3100, 3100, 18200, 18200, 11, 744, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Webwood Venomfang - In Combat - Cast \'744\'');
+
+-- Creating a new SmartAI script for [Creature] ENTRY 2001 (name: Giant Webwood Spider)
+
+-- Table creature_template
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 2001;
+
+-- Table smart_scripts
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 2001);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2001, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 6751, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Giant Webwood Spider - On Aggro - Cast \'6751\''),
```

Comment:
Spell 6751 (Weak Poison) cannot be cast directly, please use 6752 (Weak Poison Proc) on the spider instead:
```suggestion
(2001, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 11, 6752, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Giant Webwood Spider - On Respawn - Cast \'6752\''),
```